editor/store/effects.js

```diff
@@ -313,16 +337,11 @@ export default {
 			return;
 		}
 
-		if ( isCurrentPostPublished( state ) ) {
-			// TODO: Publish autosave.
-			//  - Autosaves are created as revisions for published posts, but
-			//    the necessary REST API behavior does not yet exist
-			//  - May need to check for whether the status of the edited post
-			//    has changed from the saved copy (i.e. published -> pending)
+		if ( ! isEditedPostDirty( state ) ) {
```

Comment:
With this, could we just eliminate the previous condition?

```js
if ( ! isEditedPostNew( state ) && ! isEditedPostDirty( state ) ) {
```

Reply:
yes, removed in https://github.com/WordPress/gutenberg/pull/6257/commits/6e7c57c2a9956e51d995e3e4737ffe286c400c2a